3-transport/translations/README.bn.md

```diff
@@ -0,0 +1,24 @@
+# ফার্ম থেকে ফ্যাক্টরি তে পরিবহন - আইওটি দ্বারা খাদ্য সরবরাহ ট্র্যাক করা
+
+অনেক কৃষকই বিক্রয় করার জন্য খাদ্য উৎপাদন করে - হয় তারা বাণিজ্যিক কৃষক যারা তাদের উত্থিত সমস্ত কিছু বিক্রি করে বা তারা জীবিকা নির্বাহী কৃষক যারা তাদের প্রয়োজন এর অতিরিক্ত অংশ বিক্রি করে। একরকম খাবার খামার থেকে ভোক্তার কাছে পৌঁছাতে হবে এবং এটি সাধারণত খামার থেকে হাব বা প্রসেসিং প্লান্ট, তারপরে স্টোরগুলিতে পরিবহনের উপর নির্ভর করে। উদাহরণস্বরূপ, একজন কৃষক টমেটো সংগ্রহ করবেন, বাক্সগুলিতে প্যাক করবেন, বাক্সগুলিকে একটি ট্রাকের মধ্যে লোড করবেন এবং প্রসেসিং প্ল্যানেটে পৌঁছে দেবেন। টমেটোগুলি এর পরে বাছাই করা হবে এবং সেখান থেকে প্রক্রিয়াজাত খাবার, খুচরা বিক্রয়, বা রেস্তোঁরাগুলিতে গ্রাহকদের কাছে সরবরাহ করা হবে
+
+আইওটি ডিভাইস এর মাধ্যমে ট্রানজিটেও খাবারগুলি ট্র্যাক করা যায়, যা এই সরবরাহ শৃঙ্খলায় (supply chain) সহায়তা করতে পারে - চালকদের কোথায় যাওয়া উচিত তা নিশ্চিত করা, যানবাহনের অবস্থানগুলি পর্যবেক্ষণ করা এবং যানবাহনগুলি পৌঁছানোর সময় সতর্কতা অবলম্বন করা যাতে খাবারটি দ্রুত আনলোড করা যায় এবং যত তাড়াতাড়ি সম্ভব প্রক্রিয়াজাতকরণের জন্য প্রস্তুত করা সম্ভব হয়।
+
+> 🎓  *সরবরাহ শৃঙ্খলা* বা *Supply chain*  হলো কিছু তৈরি এবং সরবরাহ করার ক্রিয়াকলাপ। উদাহরণস্বরূপ, টমেটো চাষের ক্ষেত্রে, সাপ্লাই চেইনে বীজ, মাটি, সার এবং জলের সরবরাহ, একটি কেন্দ্রীয় হাবের দ্বারা টমেটো বিতরণ, সুপার মার্কেটের স্থানীয় হাবগুলিতে পরিবহন, পৃথক সুপার মার্কেটে পরিবহন, প্রদর্শনীতে প্রদর্শন করা, তারপরে বিক্রি করা হয় একজন ভোক্তার কাছে যিনি তা খেতে বাড়িতে নিয়ে যান। প্রতিটি পদক্ষেপই এখানে সাপ্লাই চেইনের অংশ।
+
+> 🎓 সাপ্লাই চেইনের পরিবহণের অংশকে বলা হয় *logistics* ।
+
+এই চারটি পাঠে, আমরা দেখবো কীভাবে খাদ্য (ভার্চুয়াল) ট্রাকে বোঝাই করে খাবারের তদারকির মাধ্যমে সরবরাহের চেইনের উন্নতি করতে ইন্টারনেট অফ থিংস প্রয়োগ করা যায়, যেখানে যানবাহনের গন্তব্যস্থলে পৌঁছানো  পর্যন্ত ট্র্যাক করা হয়। আমরা জিপিএস ট্র্যাকিং, জিপিএস ডেটা কীভাবে সংরক্ষণ এবং ভিজ্যুয়ালাইজ করব এবং কীভাবে কোন ট্রাক তার গন্তব্যস্থলে পৌঁছালে ঠিকমতো তার ব্যাপারে জানানো যায় তা শিখবো।
+
+> 💁  এই লেসনগুলোতে আমরা কিছু ক্লাউড রিসোর্স ব্যবহার করবো। এখানে সবগুলো প্রজেক্ট সম্পূর্ণ না হলেও, আমাদেরকে [প্রজেক্ট ক্লীন-আপ](../translations/clean-up.bn.md) করতে হবে।
+
+## পাঠসমূহ
+
+1. [লোকেশন ট্র্যাকিং](lessons/1-location-tracking/translations/README.bn.md)
```

Comment:
Are these links correct? Shouldn't they be up one level?

Reply:
Thanks. They have been raised one level up.